Makefile

```diff
@@ -124,11 +127,11 @@ debug: _print_vars _generate_related_images_env
 	WATCH_NAMESPACE=$(NAMESPACE) dlv debug --listen=:2345 --headless=true --api-version=2 ./main.go --
 
 ### install: Install CRDs into a cluster
-install: manifests _kustomize
+install_crds: manifests _kustomize _init_devworkspace_crds
```

Comment:
Need to also update doc line above (`### install`)

Reply:
but then where will be confusion point :smile: 

Good catch. Will fix